$NetBSD: patch-cl,v 1.12 2017/07/03 17:42:45 prlw1 Exp $

- Leave gdbus-2.0/codegen sudirectory to devel/gdbus-codegen package.
- Don't compile tests as they include gdbus-codegen tests.
- Don't install completion data.
(patch-an)
--- gio/Makefile.am.orig	2017-06-22 12:53:56.000000000 +0000
+++ gio/Makefile.am
@@ -1,6 +1,6 @@
 include $(top_srcdir)/glib.mk
 
-SUBDIRS = gdbus-2.0/codegen
+SUBDIRS = 
 
 if OS_UNIX
 if !OS_COCOA
@@ -244,7 +244,7 @@ platform_libadd += win32/libgiowin32.la
 platform_deps += win32/libgiowin32.la
 endif
 
-SUBDIRS += . tests
+SUBDIRS += .
 
 if HAVE_FAM
 SUBDIRS += fam
@@ -898,14 +898,6 @@ gapplication_LDADD = libgio-2.0.la 			\
 endif
 endif
 
-completiondir = $(datadir)/bash-completion/completions
-completion_DATA = \
-	completion/gapplication				\
-	completion/gdbus				\
-	completion/gsettings				\
-	completion/gresource
-EXTRA_DIST += $(completion_DATA)
-
 # ------------------------------------------------------------------------
 # gresource tool
 
